app/templates/components/profile-image.hbs

```diff
@@ -0,0 +1,6 @@
+{{!- If the user has a profile image, display it. else, display a placeholder image}}
+{{#if profileImage}}
+	<img src={{profileImage}} height={{height}} width={{width}}/>
+{{else}}
+	<img src="http://via.placeholder.com/50x50" height={{height}} width={{width}}/>
```

Comment:
True! I like that! it would ensure that a 50x50 image wasn't getting stretched to fill a larger area.